todo/src/App.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import './App.css';
+import { connect } from 'react-redux';
```

Comment:
✏️ Basics Refresher: `connect()` allows us to specify which data we are listening to (through mapStateToProps), and which component we are providing the data.